src/client_networking.rs

```diff
@@ -55,49 +55,23 @@ impl From<StrictError> for ConnectionError {
 }
 
 
-// I'd change the declaration to: request_table(table_name: &str, server_address: &str)
-// Agree with name => table_name but this gets a csv. Should be called download_csv, though, to be consistent with server()
 pub fn request_csv(name: &str, address: &str) -> Result<StrictTable, ConnectionError> {
-    // Validating the server_address somehow and providing a descriptive error would probably be a good idea
```

Comment:
I'm removing this one because it was more of a thought regarding general practice, not necessarily something to preoccupy yourself with now

Reply:
The question mark operator at the end of the line takes care of that. If the address is invalid, it will return a io error.